File: src/main/java/de/thatsich/common/module/entity/AEntity.java
package de.thatsich.common.module.entity;

import net.minecraft.entity.Entity;
import net.minecraft.world.World;

/**
 * @author thatsIch
 * @date 22.03.2014.
 */
public abstract class AEntity extends Entity implements IEntity
{
	protected AEntity ( final World world )
	{
		super( world );
	}
}

File: src/main/scala/de/thatsich/minecraft/core/module/entity/AEntity.scala
package de.thatsich.minecraft.core.module.entity

import net.minecraft.entity.Entity
import net.minecraft.world.World

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
abstract class AEntity( world: World )
	extends Entity( world )
	        with IEntity
{

}
